.github/workflows/test-enos-scenario-ui.yml

```diff
@@ -100,15 +99,16 @@ jobs:
         if: contains(${{ github.event.repository.name }}, 'ent')
         run: echo "${{ secrets.VAULT_LICENSE }}" > ./enos/support/vault.hclic || true
       - name: Install Chrome Dependencies
-        if: ${{ !needs.get-metadata.outputs.chrome-installed }}
+        if: ${{ needs.get-metadata.outputs.vault_edition == 'ent' }}
         run: |
           sudo apt update
           sudo apt install -y libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev libasound2
       - name: Install Chrome
-        if: ${{ !needs.get-metadata.outputs.chrome-installed }}
+        if: ${{ needs.get-metadata.outputs.vault_edition == 'ent' }}
```

Comment:
same suggestion as above to check for both conditions

Reply:
updated to use a check that installs chrome only if it is not already installed.